frontend/src/components/Login.tsx

```diff
@@ -10,11 +11,17 @@ const tailLayout = {
 };
 
 const Login = () => {
-    const onFinish = (values: any) => {
-        console.log('Success:', values);
+    const onFinish = async (values: any) => {
+        if(!values.confirmed) {
+            console.log('Success:', values);
+        }
+        const users = await axios.get("http://localhost:3000/users", values)
+        console.log(users)
+
+        await axios.put("http://localhost:3000/users/604a8ec7b3714495913f00d8", values)
     };
 
-    const onFinishFailed = (errorInfo: any) => {
+    const onFinishFailed = async (errorInfo: any) => {
```

Comment:
This function just logs the info, remove `async` keyword as it does no asynchronous stuff.